templates/zerver/help/upload-and-share-files.md

```diff
@@ -2,28 +2,45 @@
 
 Zulip allows you to easily attach files, including images, sound, and video by:
 
-- *Clicking the Paperclip Icon* Use the paperclip icon (![paperclip](/static/images/help/paperclip.png)) located under the message field.
```

Comment:
For example: `**Clicking the Paperclip Icon** -` (also include a dash after the bold text)

Same goes for **Dragging** in the next line.